isis/src/base/apps/csminit/csminit.xml

```diff
@@ -2,17 +2,32 @@
 
 <application name="csminit" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
   <brief>
-    Attach a CSM Model state to a cube
+    Attach a CSM model state to a cube.
   </brief>
 
   <description>
     <p>
-      This program takes an ISD or state string and attempts to create
-      a valid CSM Model from the loaded CSM Libraries. Then, it attaches the
-      state string from the CSM Model to the cube for later use. If there are
-      multiple valid CSM Models that can be instantiated, then this program
-      will error and the user will have to re-run it with a specific model
-      and/or plugin name.
+      This program takes an ISD and attempts to create a valid CSM model from
```

Comment:
Expand first instance of potentially user-unfamiliar acronym "ISD" like "Instrument Support Data (ISD)" Potentially do the same with CSM!